lib/indexer/links_lookup.rb

```diff
@@ -83,6 +83,12 @@ def find_links(content_id)
           }
         )
         raise Indexer::PublishingApiError
+      rescue GdsApi::HTTPNotFound => e
+        # If the Content ID no longer exists in the Publishing API, there isn't really much
+        # we can do at this point. There doesn't seem to be any compelling reason to record
+        # this in Sentry as there is no bug to fix.
+        @logger.error("HTTP not found error fetching expanded links for #{content_id}: #{e.message}")
+        raise Indexer::PublishingApiError
```

Comment:
What is the implication of raising `Indexer::PublishingApiError`, does this mean that this task will be retried?

Reply:
I would expect so. I didn't notice this but I think it should just swallow the exception here?

Reply:
Hmm, oh yeah, I guess it might be retried... I put that in because it's what the other `rescue` blocks do and [it doesn't get sent to Sentry](https://github.com/alphagov/rummager/blob/master/lib/rummager/config.rb#L24). Let me see if I can replace it with something else.